core/controllers/reader_test.py

```diff
@@ -104,6 +107,55 @@ def test_published_explorations_are_visible_to_logged_in_users(self):
         self.assertEqual(response.status_int, 200)
 
 
+class ClassifyHandlerTest(test_utils.GenericTestBase):
+    """Test the handler for classification."""
+
+    def setUp(self):
+        """Before the test, create an exploration_dict."""
+        super(ClassifyHandlerTest, self).setUp()
+
+        #Reading YAML exploration into a dictionary
+        yaml_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                 '../tests/data/string_classifier_test.yaml')
+        with open(yaml_path, 'r') as yaml_file:
+            self.yaml_dict = yaml.load(yaml_file)
+
+        self.login(self.VIEWER_EMAIL)
+
+
+    def test_classification_handler(self):
+        """Test the classification handler."""
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+
+        # Load demo exploration.
+        exp_id = '0'
```

Comment:
If you define exp_id you might as well use it below, rather than continue hardcoding '0'?